app/actions/local/draft.test.ts

```diff
@@ -244,6 +259,44 @@ describe('updateDraftPriority', () => {
     });
 });
 
+describe('switchToGlobalDrafts', () => {
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    it('should emit navigation event on tablet', () => {
+        (isTablet as jest.Mock).mockReturnValue(true);
+        const emitSpy = jest.spyOn(DeviceEventEmitter, 'emit');
+
+        switchToGlobalDrafts();
+
+        expect(emitSpy).toHaveBeenCalledWith(Navigation.NAVIGATION_HOME, Screens.GLOBAL_DRAFTS, {});
+    });
+
+    it('should call goToScreen on non-tablet', () => {
+        (isTablet as jest.Mock).mockReturnValue(false);
+        const {goToScreen} = require('@screens/navigation');
```

Comment:
Any reason for the require instead of a normal import?

Reply:
Fix the imports